### PR TITLE
Improve AWS install instructions for CCIE, editorial and restructuring

### PR DESCRIPTION
**Pull request by michelle-luna**
Please review for accuracy and omissions, this is an editorial pass with some restructuring and a new title.

### CODE REVIEW

File: jekyll/_ccie/aws.md [hunk 1]
```diff
@@ -1,69 +1,76 @@
...
 
-(If you prefer to do this process manually or with different tools please see the [Manual AWS Instructions]({{site.baseurl}}/enterprise/aws-manual/).)
+- The private subnet for builder boxes must be a network address translation (NAT) instance or an internet gateway configured for the outbound traffic to the internet.
+- The [VPC Endpoint for S3](https://aws.amazon.com/blogs/aws/new-vpc-endpoint-for-amazon-s3/) should be enabled. Enabling the VPC endpoint for S3 should significantly improve S3 operations for CircleCI and other nodes within your subnet.
+- Adequately power the NAT instance for heavy network operations.  Depending on the specifics of your deployment, it is possible for NAT instances to become constrained by highly parallel builds using Docker and external network resources.  A NAT that is inadequate could cause slowness in network and cache operations.
+  - If you are integrating with https://github.com/, ensure that your network access control list (ACL) whitelists github.com webhooks.  When integrating with GitHub, either set up CircleCI in a public subnet, or set up a public load balancer to forward github.com traffic.
```
Comment (drazisil) [inline jekyll/_ccie/aws.md]: I think the ACL needs to be more open so the services/build can download build images, but I'd double-check with someone else.

File: jekyll/_ccie/aws.md [hunk 2]
```diff
@@ -1,69 +1,76 @@
...
-1. Use a self-signed certificate ( or if you already have one feel free to use it here )
-1. Upload your license
-1. Secure the console with a password
-1. Fill out the settings form
-1. Save and start the app (all other default settings are fine for now). You will be redirected to the System Console Dashboard,
-where you should soon see an indication that the app has started like below.
+## Next Steps for Getting Started
 
-<img src="{{site.baseurl}}/assets/img/docs/started.png" alt="Look For 'Open'" width="150" style="margin: 10px; margin-left: 200px">
+1. Click the Open link in the dashboard to go to the CircleCI Enterprise app. The Starting page for a few minutes to as the CircleCI application is booting up, then automatically redirects to the homepage.
```
Comment (drazisil) [inline jekyll/_ccie/aws.md]: Can we add a note here that the first user to access the CircleCI app becomes the admin?

File: jekyll/_ccie/aws.md (hunk 1 above)
Comment (michelle-luna) [inline jekyll/_ccie/aws.md]: thanks, I added a comment to the doc about this

File: jekyll/_ccie/aws.md (hunk 2 above)
Comment (michelle-luna) [inline jekyll/_ccie/aws.md]: thanks, added

**Pull request merged**